### DISCUSSION TITLE
Having couple of queries related to Workload Discovery on AWS

### DISCUSSION THREAD

**Discussion by gokendra1** [aws-solutions/workload-discovery-on-aws#558] [General]
[Raising this on customer's behalf]

Hello Team,

Can you please address the following queries related to Workload Discovery on AWS:

1. Is it capable of displaying the architecture of all accounts in a single diagram?
2. Can I select which services to include in the diagram?
3.  What information does AWS Perspective provide about Transit Gateways (TGWs)? Specifically, does it show TGWs and their connections across VPCs,

**Comment by svozza** [comment]
1. No, we do not have the option to do this. The main reason is that a diagram such as that will be visually overwhelming but also that a diagram with thousands of resources on it will likely crash the browser tab.
2. Yes, this is documented [here](https://aws-solutions.github.io/workload-discovery-on-aws/workload-discovery-on-aws/2.0/WebUI-Features.html#_filters).
3. We support the following transit gateway relationhips:

`AWS::EC2::TransitGatewayAttachment` -> `AWS::EC2::TransitGateway`
`AWS::EC2::TransitGatewayAttachment` -> `AWS::EC2::VPC`
`AWS::EC2::TransitGatewayAttachment` -> `AWS::EC2::Subnet`
`AWS::EC2::TransitGateway` -> `AWS::EC2::TransitGatewayRouteTable`

**Comment by aws-ymson** [comment]
Closing this out: Workload Discovery on AWS has been deprecated as of 08/14/2026 and is no longer being maintained.
